### ISSUE TITLE
Doesn't work if there is more than 1 form

### ISSUE THREAD

**Issue by kingzfate** [TimothyMeadows/reCAPTCHA.AspNetCore#69]
If there is more than 1 form with Recaptcha on the page, then all methods that are written in the forms are called.
For example, on my site there is a registration page, I want to have a Recaptcha there, also in the header I have a tab with an entrance to the site, and there I want to have a Recaptcha. As a result, both Register and Login will be called in the controller. If you choose id then it does not help.
 @(Html.Recaptcha<RecaptchaV2Invisible>(RecaptchaSettings?.Value, new RecaptchaV2Invisible { Id = "registr"}))
 @(Html.Recaptcha<RecaptchaV2Invisible>(RecaptchaSettings?.Value, new RecaptchaV2Invisible { Id = "login"}))

**Comment by TimothyMeadows** [comment]
Looks like exposure for Uid might need to be added to the model for v2 invis. This will be added for the next version when time permits. The reason you can't use it on more than 1 form is because the dynamic methods that get written would always be the same. Will need Uid, and Id both. This is a bug.

**Comment by kingzfate** [comment]
RecaptchaV3Invisible Also does not work(

**Comment by TimothyMeadows** [comment]
I will be working to address this, this weekend.

**Comment by TimothyMeadows** [comment]
Turns out this is not an actual bug. In a previous version (2.x core version) this had existed as such which is what confused me. However i had already solved this by 3.x. I have added an example of 2 forms on the page (sharing the same model) working just fine each with there own recaptcha token respectively. It does seems MVC it's self has some complications with processing more than 1 form in general but this is not related to this library.

https://github.com/TimothyMeadows/reCAPTCHA.AspNetCore/blob/master/reCAPTCHA.AspNetCore.Example/Views/Home/Contact4.cshtml

Take note that you need to specific a new Id in the the model for each form beyond the first. This Id must match the button id for the form.

**Comment by TimothyMeadows** [comment]
Going to close this for now, please re-open if you think this is in error.

**Comment by kingzfate** [comment]
Yes, I did exactly the same for the form RecaptchaV2Invisible. But I would like to use RecaptchaV3HiddenInput. And I do not know how to separate these calls there since there is no param "Id"
